### PR TITLE
Parallexaxis doc

### LINKED ISSUE #141: Allow changing the figure size of a `parcoords.ParallelAxes()` plot
Currently for a [`parcoords.ParallelAxes()`](https://emaworkbench.readthedocs.io/en/latest/ema_documentation/analysis/parcoords.html) you can change the number format, fontsize and label rotation, but not the figure size (or aspect ratio). If you have many variables this can be desirable.

A maybe even broader approach to this issue would be returning a [`matplotlib.pyplot.figure`](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.figure.html) instead of a `NoneType` when calling `ParallelAxes.plot()`, for which you can change it's size and other aspects afterwards (with [`Figure.set_size_inches()`](https://matplotlib.org/stable/api/figure_api.html#matplotlib.figure.Figure.set_size_inches)).

### PR DESCRIPTION
**Pull request by quaquel**
closes #141

### CODE REVIEW

File: ema_workbench/analysis/parcoords.py [hunk 1]
```diff
@@ -143,6 +143,27 @@ class ParallelAxes:
     rot : float, optional
           rotation of axis labels
 
+
+    Attributes
+    ----------
+    limits : DataFrame
```
Comment (EwoutH) [inline ema_workbench/analysis/parcoords.py:149]: Add a description of `limits`

File: ema_workbench/analysis/parcoords.py [hunk 2]
```diff
@@ -143,6 +143,27 @@ class ParallelAxes:
...
+    recoding : dict
+               non numeric columns are converting to integer variables
+    flipped_axes : set
+                   set of Axes that are to be shown flipped
+    axis_labels : list of str
+    fontsize : int
+    normalizer : MinMaxScaler instance
```
Comment (EwoutH) [inline ema_workbench/analysis/parcoords.py:156]: Add a description of `normalizer`

File: ema_workbench/analysis/parcoords.py [hunk 3]
```diff
@@ -143,6 +143,27 @@ class ParallelAxes:
...
+    fig : a matplotlib Figure instance
+    axes : list of matplotlib Axes instances
+    ticklabels : list of str
+    datalabels : list of str
+                 labels associated with lines
+
+
+    The basic setup of the Parallel Axis plot is a row of mpl Axes instances, with all whitespace
```
Comment (EwoutH) [inline ema_workbench/analysis/parcoords.py:164]: Should this line have a header, since from here class methods are discussed?

**Review by EwoutH — CHANGES_REQUESTED** [review]
It looks like there is overlap between the _Parameters_ and _Attributes_ sections. I don't think this is ideal. See also the [readthedocs build](https://emaworkbench--249.org.readthedocs.build/en/249/ema_documentation/analysis/parcoords.html#ema_workbench.analysis.parcoords.ParallelAxes).

**Comment by quaquel** [comment]
I can't figure out how to discard that change. It indeed should not be there. I'll fix the rest.

Note that the attributes and parameters can overlap. Parameters documents the arguments for the `__init__` while attributes documents the state of a class.

File: ema_workbench/analysis/parcoords.py (hunk 1 above)
Comment (quaquel) [reply ema_workbench/analysis/parcoords.py:148]: added the description from the parameters part

File: ema_workbench/analysis/parcoords.py (hunk 2 above)
Comment (quaquel) [reply ema_workbench/analysis/parcoords.py]: I made this an implementation detail and removed it from the documentation. It is not part of the public API so you don't want to mess with the normalizer.

File: ema_workbench/analysis/parcoords.py (hunk 3 above)
Comment (quaquel) [reply ema_workbench/analysis/parcoords.py:166]: moved it into Notes which is the proper place for implementation details

**Review by EwoutH — APPROVED** [review]

**Comment by EwoutH** [comment]
Looks good!

Please squash while merging and cleanup the commit message and body while merging.

**Pull request merged**